p2p/README.md

```diff
@@ -40,102 +30,36 @@ Research: TCP hole punching with the help of bootnodes to allow inbound connecti
-
-Additionally, once every 5 minutes, the peers shall exchange peer information they've learned within the last 5 minutes. This means that the peer which initiated the connection sends a `Pex` message which contains either an empty list (if it hasn't discovered new nodes) or a list of nodes it has discovered within the last 5 minutes. The other node responds to this message with a `PexAck` message that also contains either an empty list or a list of new nodes discovered within the last 5 minutes.
+Once a minute, the node will check if the remote peer is still active by sending a `PingRequest` message. The peer that receives a `PingRequest` must respond with a `PingResponse` message. If no response is heard, the connection is closed. `PingRequest`/`PingResponse` are exchanged only if there has been no activity on the socket in the last minute. Each `PingRequest` message contains a 64-bit nonce that the corresponding `PingResponse` sends back to ascertain that a correct `PingRequest` message was acknowledged.
 
-In addition to maintaining each connection individually by exchanging `Ping`/`Pong` and `Pex`/`PexAck` messages, the Mintlayer node must also maintain the overall set of connections by accepting incoming connections/establishing new outbound connections to maintain the total amount of connections at the specified number which is by default set to 32. This means that the event loop of the node must reject new connections if the number of active connections is higher than it should be or if the number of connections is lower than the lower bound for number of active connections, the node must try establishing new connections with peers it has learned from previous connections. In case there aren't 32 peers in the network, the node must try to connect to as many nodes as it can as the number of nodes in the network grows, it must try to connect to the new nodes until it reaches 32 active connections.
+In addition to maintaining each connection individually, the Mintlayer node will also maintain the overall set of connections by accepting incoming connections/establishing new outbound connections to maintain the total amount of connections at the specified number. This means that the event loop of the node will reject new connections if the number of active connections is higher than it should be; if the number of connections is lower than the lower bound for number of active connections, the node will try establishing new connections with peers it has learned from previous connections.
 
 ### Message exchange
```

Comment:
Kind of excessive to erase just everything... but I don't care right now.

Some of the frames below are correct.

Reply:
I've rewritten and added those sections back.